src/lib/components/wizard-completion-step.component.spec.ts

```diff
@@ -24,7 +24,7 @@ import {WizardComponent} from './wizard.component';
 })
 class WizardTestComponent {
 
-  @ViewChild(WizardComponent)
+  @ViewChild(WizardComponent, {static: false})
```

Comment:
For now I added to all `@ViewChild` and `@ContentChild` decorators `{static: false}`. I'm not sure if this is the best configuration for all our `@XChild` decorators or if there are some where it makes more sense to use `{static: true}` instead. What do you think @earshinov?

Reply:
@madoar , Using `{ static: false }` by default is the right choice. One of the reasons is that it will be the default in Angular 9. There is more information is the official guide: https://angular.io/guide/static-query-migration#how-do-i-choose-which-static-flag-value-to-use-true-or-false